<launch>
  <arg name="joy_config" default="warthog" />
  <arg name="config_filepath" default="$(find doughnut_calib)/config/$(arg joy_config).config.yaml" />

  <node pkg="doughnut_calib" name="doughnut_calib" type="doughnut_calib.py" output="screen">
    <rosparam command="load" file="$(arg config_filepath)" />
    <remap from="joy_in" to="joy"/>
    <remap from="cmd_vel_out" to="cmd_vel"/>
  </node>

  <!-- node pkg="rosbag" type="record" args="-a" name="record" output="screen"/ -->

</launch>
